packages/openapi-generator/README.md

```diff
@@ -1,39 +1,89 @@
-## OpenApi Client Generator
+## SAP Cloud SDK OpenAPI Client Generator (Beta)
 
-This is currently not release ready. excluded in `mail.ts`.
+Generate custom JavaScript/TypeScript clients for OpenAPI services.
```

Comment:
I would say it are services with OpenAPI definitions, but perhaps I am too picky here.